spree_multi_domain.gemspec

```diff
@@ -17,13 +17,14 @@ Gem::Specification.new do |s|
   s.require_path = 'lib'
   s.requirements << 'none'
 
-  version = '>= 3.1.0', '< 4.0'
+  version = '>= 3.1.0', '< 5.0'
   s.add_dependency 'spree_core', version
   s.add_dependency 'spree_backend', version
   s.add_dependency 'spree_frontend', version
   s.add_dependency 'spree_api', version
   s.add_dependency 'spree_extension'
   s.add_dependency 'spree_analytics_trackers'
+  s.add_dependency 'paperclip'
 
   s.add_development_dependency 'capybara'
```

Comment:
@PSzyszkaSpark we also need to add `sass-rails` to dev dependencies for Spree 3.7+